_drafts/oryx/orchestration/app-plugins.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Application plugins
+description: Plugins of the Oryx Application
+template: concept-topic-template
+---
+
+# Application plugins
+
+When you create an Oryx Application with the `appBuilder()` function, it creates an instance of `App`. The `App` is a a shell that can be enhanced with (custom) plugins (`AppPlugin`). Plugins allow you to extend Oryx core behaviour without modifying the core code of the framework.
```

Comment:
Is `AppPlugin` an example or a type?

Reply:
Can `App` be enhanced with regular plugins as opposed to custom ones?

Reply:
Yes `AppPlugin` is a type.
`App` is in fact enhanced by built-in plugins and can be enhanced with custom ones as well.